src/Appwrite/Platform/Tasks/QueueCount.php

```diff
@@ -1,57 +0,0 @@
-<?php
-
-namespace Appwrite\Platform\Tasks;
-
-use Utopia\CLI\Console;
-use Utopia\Platform\Action;
-use Utopia\Queue\Client;
-use Utopia\Queue\Connection;
-use Utopia\Validator\Text;
-use Utopia\Validator\WhiteList;
-
-class QueueCount extends Action
-{
-    public static function getName(): string
-    {
-        return 'queue-count';
-    }
-
-
-    public function __construct()
-    {
-        $this
-            ->desc('Return the number of from a specific queue identified by the name parameter with a specific type')
-            ->param('name', '', new Text(100), 'Queue name')
-            ->param('type', '', new WhiteList([
-                'success',
-                'failed',
-                'processing',
-            ]), 'Queue type')
```

Comment:
We added the size() method back right? So we should be able to still get these metrics?